spec/models/rent_spec.rb

```diff
@@ -11,4 +11,11 @@
   it { is_expected.to validate_presence_of :to }
 
   it { expect(rent.from).to be <= rent.to }
+
+  it '#validate_date_from_is_less_than_date_to' do
```

Comment:
Check what I said before about starting all examples with a verb! If you want to specify what's the method you are testing, use a `describe`. For example
```ruby
describe '#my_method'
  it 'returns X' do

  end
end
```